mlir/include/mlir/Dialect/LLVMIR/NVVMOps.td

```diff
@@ -422,6 +422,22 @@ def NVVM_SetMaxRegisterOp : NVVM_PTXBuilder_Op<"setmaxregister"> {
   let hasVerifier = 1;
 }
 
+def NVVM_FenceMbarrierInitOp : NVVM_PTXBuilder_Op<"fence.mbarrier.init"> {
```

Comment:
I would like to get your thoughts on:
Should we name the Op as "fence.mbarrier_init" ?
(All other places below use the "_" version).

Either ways, change looks good to me.

Reply:
As far as I can see, MLIR prefers '_' more than '.' . I think I need to change all the OPs to that. I'll do that in follow up PR. It's great that you catch this. So we will have the consistency.

Reply:
Sure, thank you!